resources/appsync-graphqlapis.go

```diff
@@ -0,0 +1,76 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/appsync"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+// AppSyncGraphqlApi - An AWS AppSync GraphQL API
+type AppSyncGraphqlApi struct {
```

Comment:
```suggestion
type AppSyncGraphqlAPI struct {
```

Reply:
Absolutely. What about the resource-type (as in the config) casing?

Reply:
Are you referring to `apiID`? That would be fine by golangs conventions. In general all identifiers/names should adhere to golang conventions. Thanks again.

Reply:
As in the registered resource name `"AppSyncGraphqlApi"` that will appear to and be referenced by a user.

Reply:
Ah, you are right! That should also match the golang convention and should be similar to the type :) Thanks!

Reply:
I've push the commit and updated the example config and output in the PR description above to reflect it. Thank you again.